modules/pulsestorm/magento2/cli/generate/module/module.php

```diff
@@ -9,6 +9,18 @@
 pestle_import('Pulsestorm\Xml_Library\addSchemaToXmlString');
 pestle_import('Pulsestorm\Cli\Code_Generation\templateRegistrationPhp');
 pestle_import('Pulsestorm\Magento2\Cli\Xml_Template\getBlankXml');
+pestle_import('Pulsestorm\Magento2\Cli\Library\getModuleInformation');
+
```

Comment:
Composer packages usually have their code in a folder that's different from the package base, so we now have the concepts of a module directory (where's the code!) and a package directory (where's composer.json and registration.php)